back/engines/commercial/smart_groups/spec/rules/participated_in_project_spec.rb

```diff
@@ -56,7 +56,7 @@
     let_it_be(:user3) { create(:user) }
     let_it_be(:user4) { create(:user) }
     let_it_be(:idea1) { create(:idea, project: project1, author: user1) }
-    let_it_be(:propsal1) { create(:proposal, project: project1, author: user4) }
+    let_it_be(:propsal1) { create(:proposal, project: project1, author: user4, creation_phase: create(:proposals_phase, project: project1)) }
```

Comment:
I think proposals should not (normally) have a `creation_phase` set. I think you are setting this purposefully to test it is filtered correctly as an 'abnormal' proposal, correct?

If so, I'd think about explaining why a creation_phase is set in a comment, to help other devs understand this special case.

Reply:
Ahh, got it. Thanks for explaining.